### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by neetukumari4858**
PlayList Functionality Added
On the Playlist Page
- I can create a new playlist.
- I can delete an existing playlist
- I can remove a video from a particular playlist.

### CODE REVIEW

File: src/Reducers/PlaylistReducer.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    const {type,payload}=action 
+    switch (type) {
+        case "CREATE_PLAYLIST":
+            return [...state,payload.playList]
+        case "ADD_TO_PLAYLIST":
+            return state.map((playlist)=>
+            playlist._id===payload.Playlist._id? {...playlist,videos : [...playlist.videos,payload.video]}:playlist)
+        case "REMOVE_PLAYLIST":
+            const removedPlaylist = state.filter((item) => item._id !== payload)
+            console.log(removedPlaylist,"removedPlaylist")
```
Comment (Bhakti199) [inline src/Reducers/PlaylistReducer.js:11]: remove console.log

**Review by Bhakti199 — APPROVED** [review]
Thi code looks fine to me.
god to merge.

File: src/Reducers/PlaylistReducer.js (hunk 1 above)
Comment (neetukumari4858) [reply src/Reducers/PlaylistReducer.js:11]: okey I Will remove console.log

**Review by MonikaJhingan — APPROVED** [review]
The code looks good go be merged.

**Pull request merged**